.github/workflows/connector_integration_tests.yml

```
@@ -29,3 +29,22 @@ jobs:
         run: python ./tools/bin/ci_integration_workflow_launcher.py base-normalization source-acceptance-test source:alpha source:beta source:GA destination:alpha destination:beta destination:GA
         env:
           GITHUB_TOKEN: ${{ secrets.SLASH_COMMAND_PAT }}
+  launch_integration_tests_alpha_only:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v3
+      - name: Install Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: "zulu"
+          java-version: "17"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install PyYAML requests
+      - name: Launch Integration Tests
```

```suggestion
      - name: Launch Integration Tests (Alpha connectors)
```
Nit: titling the "Launch Integration Tests" job steps for the jobs more clearly